src/main/java/com/hm/achievement/lang/Lang.java

```diff
@@ -0,0 +1,41 @@
+package com.hm.achievement.lang;
+
+import com.hm.achievement.AdvancedAchievements;
+import org.apache.commons.lang3.StringUtils;
+
+public interface Lang {
```

Comment:
Could you please add a comment header with a brief description an the author, to stay consistent with other classes and help out developers navigating through the classes?